ha/setup/ha_setup.py

```diff
@@ -196,31 +198,84 @@ def process(self):
         cluster_secret = Cipher.decrypt(key, cluster_secret.encode('ascii')).decode()
         s3_instances = self._get_s3_instance(machine_id)
 
-        try:
-            self._cluster_manager = CortxClusterManager()
-            # Create cluster
-            Log.info(f"Creating cluster: {cluster_name} with node: {node_name}")
-            cluster_output: str = self._cluster_manager.cluster_controller.create_cluster(
-                cluster_name, cluster_user, cluster_secret, node_name)
-            Log.info(f"Cluster creation output: {cluster_output}")
-            # TODO: Handle race condition cluster and resource create with global check.
-            if json.loads(cluster_output).get("status") == STATUSES.SUCCEEDED.value:
-                # Put cluster in standby mode
-                standby_output: str = self._cluster_manager.node_controller.standby(node_name)
-                Log.info(f"Put node in standby output: {standby_output}")
-                if json.loads(standby_output).get("status") != STATUSES.FAILED.value:
-                    Log.info("Creating pacemaker resources")
-                    create_all_resources(s3_instances=s3_instances)
-                    Log.info("Created pacemaker resources successfully")
+        self._cluster_manager = CortxClusterManager()
+        # Check if the cluster exists already, if not, create one.
+        cluster_exists = self._confstore.key_exists(const.CLUSTER_CONFSTORE_NODES_KEY)
```

Comment:
`# Check if the cluster exists already, if not, create one.`
Put this sentences in log.info
If consul down or halt log will be useful to track

Reply:
done.